src/components/WalletManager.tsx

```diff
@@ -121,7 +125,12 @@ export default function WalletManager() {
           {verifiedAddresses.includes(profile) && <VerifiedIcon />}
         </h1>
         <p className={styles.Address}>
-          {formatAddress(profile)}
+          <Tooltip text="Check address on viewblock">
+            <a href={getViewblockLinkForAddress(profile)} target="_blank">
```

Comment:
Are you sure this is going to work? Hrefs don't work in popups, so I usually do an onclick event and open a new tab using the chrome API.

Reply:
Nevermind this is fine